.circleci/config.base.yml

```diff
@@ -36,22 +36,31 @@ executors:
     environment:
       AMPLIFY_DIR: C:/home/circleci/repo/out
       AMPLIFY_PATH: C:/home/circleci/repo/out/amplify.exe
-
-  l: &linux-e2e-executor
+# If you update this name, make sure to update it in split-e2e-tests.ts as well
```

Comment:
does push hook catch it ?

Reply:
it never has; that script only checks if this file is valid/compiles